e2e/pages/administration/jobs/scheduled/scheduled.readonly.spec.js

```diff
@@ -0,0 +1,30 @@
+import LoginComponent from '../../../../components/login/login.po';
+import ScheduledPage from './scheduled.po';
+import Hooks from '../../../../utilities/hooks';
+
+let hooks, loginComponent, scheduled;
+
+beforeEach(async () => {
+    loginComponent = new LoginComponent();
+    scheduled = new ScheduledPage();
+    hooks = new Hooks();
+    await hooks.open('#/search');
+});
+
+describe('When an ONC-Staff user is logged in', () => {
```

Comment:
```suggestion
describe('when an ONC-Staff user is logged in', () => {
```
I'd prefer to keep all of our `describe` and `it` block text lower-case. The basic reason is that these sentence fragments can be combined to make full sentences, but the nesting can change in a variety of ways, and they can change over time, and if we start with a capital letter at one point, but then nest that block later, the capital letter will show up in the middle of a sentence. It's simpler to just keep them all as lowercase, always